modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectFieldResourceImpl.java

```diff
@@ -204,9 +197,7 @@ public ObjectField putObjectField(
 		return _toObjectField(
 			_objectFieldService.updateObjectField(
 				objectField.getExternalReferenceCode(), objectFieldId,
-				ObjectFieldUtil.getListTypeDefinitionId(
-					contextUser.getCompanyId(), _listTypeDefinitionLocalService,
-					objectField),
+				objectField.getListTypeDefinitionId(),
```

Comment:
> because the user can't edit the state.

But the user can change the Picklist that is not a state..
[Gravação de tela de 24-03-2023 10:20:39.webm](https://user-images.githubusercontent.com/34186632/227532237-5fb95eda-bd62-45ec-abaf-f1290a0d24e0.webm)